examples/tokenize/tokenize.cpp

```diff
@@ -3,33 +3,259 @@
 
 #include <cmath>
 #include <cstdio>
+#include <iostream>
+#include <fstream>
 #include <string>
 #include <vector>
 
+static void print_usage_information(const char* argv0, FILE* stream) {
```

Comment:
Put spaces front and back - see the rest of the codebase:

```suggestion
static void print_usage_information(const char * argv0, FILE * stream) {
```

Reply:
Changed all the `something*`s into `something *`s in the new push 👍